src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -114,6 +117,9 @@ public sealed override ImmutableArray<Location> Locations
             }
         }
 
+        public override Location? TryGetFirstLocation()
```

Comment:
```suggestion
        public override Location TryGetFirstLocation()
```

Can't we remove the `?` here?

Reply:
yes.  i need to remember that overrides can do that.  will fix up a bunch of these.  thanks!

Reply:
done.